plugins/woocommerce/client/blocks/tests/e2e/tests/add-to-cart-with-options/add-to-cart-with-options.block_theme.spec.ts

```diff
@@ -60,6 +68,12 @@ test.describe( 'Add to Cart + Options Block', () => {
 		pageObject,
 		editor,
 	} ) => {
+		// eslint-disable-next-line playwright/no-skipped-test
+		test.skip(
+			wordPressVersion <= 6.7,
+			'Skipping test as withSyncEvent is available starting from WordPress 6.8'
+		);
+
```

Comment:
There's no need to manually call `getInstalledWordPressVersion` in the file. There's `wpCoreVersion` argument provided to tests and you can use it like this 

```suggestion
		pageObject,
		editor,
		wpCoreVersion
	} ) => {
		// eslint-disable-next-line playwright/no-skipped-test
		test.skip(
			wpCoreVersion <= 6.7,
			'Skipping test as withSyncEvent is available starting from WordPress 6.8'
		);

```

Reply:
Thank you @kmanijak! I've requested another review.